packages/panamax/panamax-init.sh

```
@@ -0,0 +1,11 @@
+# Create a vendor directory with all the intended binaries
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)